src/index.ts

```diff
@@ -806,161 +842,161 @@ export class SunsynkPowerFlowCard extends LitElement {
             </a>
             <a href="#" @click=${(e) => this.handlePopup(e, config.entities.priority_load_243)}>
-              <svg xmlns="http://www.w3.org/2000/svg" id="pbat" x="46%" y="${useautarky != false ? "65.5%" : "70%"}" width="18" height="18" viewBox="0 0 24 24"><path display="${stateObj25.state === 'off' && priority !== false ? '' : 'none'}" fill="${inverter_colour}" d="M15.95 21.175L13.1 18.35l1.425-1.4l1.425 1.4l3.525-3.525l1.425 1.4l-4.95 4.95ZM8 22q-.425 0-.713-.288T7 21V5q0-.425.288-.713T8 4h2V2h4v2h2q.425 0 .713.288T17 5v7q-.525 0-1.025.088T15 12.35V6H9v14h2.35q.2.575.488 1.075t.687.925H8Zm1-2h2.35H11h.35H9Z"/></svg>
-              <svg xmlns="http://www.w3.org/2000/svg" id="pload" x="46%" y="${useautarky != false ? "65.5%" : "70%"}" width="18" height="18" viewBox="0 0 24 24"><path display="${stateObj25.state === 'on' && priority !== false ? '' : 'none'}" fill="${inverter_colour}" d="m15 13l-4 4v-3H2v-2h9V9l4 4M5 20v-4h2v2h10v-7.81l-5-4.5L7.21 10H4.22L12 3l10 9h-3v8H5Z"/></svg>
-              <text id="priority_text_load"x="50%" y="${useautarky != false ? "68.5%" : "73%"}" class="st3 left-align" display="${stateObj25.state === 'on' && priority !== false ? '' : 'none'}" fill="${inverter_colour}">${localize('common.priority_load')}</text>
-              <text id="priority_text_batt"x="50%" y="${useautarky != false ? "68.5%" : "73%"}" class="st3 left-align" display="${stateObj25.state === 'off' && priority !== false ? '' : 'none'}" fill="${inverter_colour}">${localize('common.priority_batt')}</text>
+              <svg xmlns="http://www.w3.org/2000/svg" id="pbat" x="210" y="${useautarky != 'no' ? "251" : "268"}" width="18" height="18" viewBox="0 0 24 24"><path display="${stateObj25.state === 'off' && priority !== 'no' ? '' : 'none'}" fill="${inverter_colour}" d="M15.95 21.175L13.1 18.35l1.425-1.4l1.425 1.4l3.525-3.525l1.425 1.4l-4.95 4.95ZM8 22q-.425 0-.713-.288T7 21V5q0-.425.288-.713T8 4h2V2h4v2h2q.425 0 .713.288T17 5v7q-.525 0-1.025.088T15 12.35V6H9v14h2.35q.2.575.488 1.075t.687.925H8Zm1-2h2.35H11h.35H9Z"/></svg>
+              <svg xmlns="http://www.w3.org/2000/svg" id="pload" x="210" y="${useautarky != 'no' ? "251" : "268"}" width="18" height="18" viewBox="0 0 24 24"><path display="${stateObj25.state === 'on' && priority !== 'no' ? '' : 'none'}" fill="${inverter_colour}" d="m15 13l-4 4v-3H2v-2h9V9l4 4M5 20v-4h2v2h10v-7.81l-5-4.5L7.21 10H4.22L12 3l10 9h-3v8H5Z"/></svg>
+              <text id="priority_text_load" x="228.5" y="${useautarky != 'no' ? "262" : "280"}" class="st3 left-align" display="${stateObj25.state === 'on' && priority !== 'no' ? '' : 'none'}" fill="${inverter_colour}">Priority Load</text>
```

Comment:
Did you intentional remove the localisation of "Priority Load" and "Priority Batt"

Reply:
No i will fix that now!